xbmc/utils/log.cpp

```diff
@@ -214,7 +214,7 @@ bool CLog::WriteLogString(int logLevel, const std::string& logString)
                                   minute,
                                   second,
                                   static_cast<int>(millisecond),
-                                  (uint64_t)CThread::GetCurrentThreadId(),
+                                  static_cast<uint64_t>(CThread::GetCurrentThreadNativeHandle()),
```

Comment:
c-style cast is here needed too

Reply:
What type should `CThread::GetCurrentThreadNativeHandle` return?

Reply:
uint64_t?

Reply:
I actually like that. I suggested it in the channel. I'll put something out there tonight.